nexus/db-queries/src/authz/policy_test/mod.rs

```diff
@@ -343,19 +352,27 @@ async fn test_conferred_roles() {
         .await
         .unwrap();
 
-    // Assemble the list of resources that we'll use for testing.  This is much
-    // more limited than the main policy test because we only care about the
-    // behavior on the Fleet itself.  We also create a Silo because the
-    // ResourceBuilder will create for us users that we can use to test the
-    // behavior of each role.
     let exemptions = resources::exempted_authz_classes();
     let mut coverage = Coverage::new(&logctx.log, exemptions);
+
+    // Assemble the list of resources that we'll use for testing.  This is much
+    // more limited than the main policy test because we only care about the
+    // behavior on the Fleet itself, as well as some top-level resources that
+    // exist outside of a silo.
     let mut builder =
         ResourceBuilder::new(&opctx, &datastore, &mut coverage, main_silo_id);
     builder.new_resource(authz::FLEET);
-    builder.new_resource_with_users(main_silo).await;
+    builder.new_resource(authz::IP_POOL_LIST);
     let test_resources = builder.build();
 
+    // We also create a Silo because the ResourceBuilder will create for us
+    // users that we can use to test the behavior of each role.
+    let mut silo_builder =
+        ResourceBuilder::new(&opctx, &datastore, &mut coverage, main_silo_id);
+    silo_builder.new_resource(authz::FLEET);
+    silo_builder.new_resource_with_users(main_silo).await;
+    let silo_resources = silo_builder.build();
+
```

Comment:
Oh, because `new_resource_with_users` would add some extra resources like the silo which `authorize_everything` would pick up in addition to the just the resources we want to test. I guess since we only have the 2 right now (`Fleet` and `IpPoolList`) I could've just made the extra `authorize_one_resource` call but like this makes it a bit easier to test more in the future.